### DISCUSSION TITLE
Teleport not working, stucked at Teleport-auth step

### DISCUSSION THREAD

**Discussion by vks409** [gravitational/teleport#29205] [Q&A] [answered]
We are stuck currently in teleport installation with Argocd deployment.  The teleport is not able to get authenticated with AWS .
This is when we had the S3 teleport-bucket existing, if the bucket doesn't exist then teleport should be able to create it, but in place of creating s3 bucket it does not create it. It seems teleport does not have enough access in AWS  to create S3 even though policy exist refer here. It was able to see dynamo db policy as per step 2 but for s3 it reflect's an error.
 
https://goteleport.com/docs/deploy-a-cluster/helm-deployments/aws/ 
 
I see 2 issues but none of them has clear solution. I upgraded the teleport to latest version didn't help , totally stuck right now. 
 
https://github.com/gravitational/teleport/issues/28999
https://github.com/gravitational/teleport/issues/28744
 
Let me know if you need additional information
 
ERROR REPORT:

```
Original Error: *trace.BadParameterError AccessDenied: Access Denied
status code: 403, request id: 5HESXM7H0TBX0C9J, host id: uzzZoZ/t45KvUGhQK8ONBBClVQULZxpyeNtl0DKTJjyU2GI&#43;mj9ULDGRLpZelXbz/O48VlXJFKI=
Stack Trace:
github.com/gravitational/teleport/lib/utils/aws/s3.go:44 github.com/gravitational/teleport/lib/utils/aws.ConvertS3Error
github.com/gravitational/teleport/lib/events/s3sessions/s3stream.go:195 github.com/gravitational/teleport/lib/events/s3sessions.(*Handler).ListUploads
github.com/gravitational/teleport/lib/events/complete.go:161 github.com/gravitational/teleport/lib/events.(*UploadCompleter).CheckUploads
github.com/gravitational/teleport/lib/events/complete.go:148 github.com/gravitational/teleport/lib/events.(*UploadCompleter).Serve
runtime/asm_amd64.s:1598 runtime.goexit
User Message: AccessDenied: Access Denied
status code: 403, request id: 5HESXM7H0TBX0C9J, host id: uzzZoZ/t45KvUGhQK8ONBBClVQULZxpyeNtl0DKTJjyU2GI&#43;mj9ULDGRLpZelXbz/O48VlXJFKI=] events/complete.go:149
```
Attachment(s)

![image](https://github.com/gravitational/teleport/assets/139781720/2ba87f15-1358-42eb-8bc7-39d46ab53372)

**Comment by webvictim** [comment, marked as answer]
Please fix the issue in your related discussion (https://github.com/gravitational/teleport/discussions/29204#discussion-5411632) and then reopen this if it's still an issue.
